opentelemetry/proto/metrics/v1/metrics.proto

```diff
@@ -482,42 +480,59 @@ message DoubleHistogramDataPoint {
   // buckets if a histogram is provided.
   double sum = 5;
 
-  // bucket_counts is an optional field contains the count values of histogram
-  // for each bucket.
+  // buckets are one of several defined bucketing strategies
   //
-  // The sum of the bucket_counts must equal the value in the count field.
+  // If this field is omitted, there is an implied single bucket
+  // spanning (-Inf,+Inf) containing the whole population.
   //
-  // The number of elements in bucket_counts array must be by one greater than
-  // the number of elements in explicit_bounds array.
-  repeated fixed64 bucket_counts = 6;
+  // Each of the supported buckets in the oneof below are expected
+  // to state under which circumstances buckets of that type are
+  // considered Mergeable in a non-lossy way with buckets of the same
+  // type.
+  //
+  // When a single Metric uses Histograms with buckets of that are not
+  // mergeable, they will pass through OpenTelemetry collectors as
+  // distinct Metric series, as they cannot be merged.
```

Comment:
Nit: Grammar (drop `of`)

```
When a single Metric uses Histograms with buckets that are not mergeable, they will pass through OpenTelemetry collectors as distinct Metric series, as they cannot be merged.
```

Second nit:

> Pass through OpenTelemetry collectors as distinct metric series

I assume this means in the context of OTLP.   A backend exporter should be free to do whatever they think best for their users and backend, but OTLP will guarantee separate series being pushed.

I think a better way to phrase this:

```
OTLP does not allow two `Metric` instances having `Histogram`s with unmergable bucket types to be merged.  These will remain disjoint `Metric` instances through any processors within OpenTelemetry.

Reply:
Done.